### ISSUE TITLE
cmip6_tile cannot download data because geodata server is down for maintenance

### ISSUE THREAD

**Issue by ManuelSpinola** [rspatial/geodata#61]
I am trying to use the function cmip6_tile  but it say that the geodata server is down for maintainance.

p <- cmip6_tile(lon = -84.2, lat = 9.6, model = "BCC-CSM2-MR", ssp = "585", time = "2041-2060", var = "bioc", path=tempdir())

The geodata server is down for maintenance. 
It is expected to be back online on October 12, 2023.
download failed

**Comment by tubabucak** [comment]
Yes I am also having the similar issues though cmip6_world function is working.

**Comment by Raksha-95** [comment]
I am having the same issue. The function cmip6_world is not working for me 

fut.clim <- geodata::cmip6_world(model="GISS-E2-1-G", ssp="585", time="2041-2060", var="bioc", res=5, path=tempdir())

The geodata server is down for maintenance. 
It is expected to be back online on October 12, 2023

**Comment by tubabucak** [comment]
I think it is a temporary issue. I was able to download the data last week with cmip6_world but know I am also having the same error that you got when I run the same code.

**Comment by NCBaso** [comment]
it is now the end of 2024 and i am experiencing the same issue:  biof=cmip6_tile("CNRM-CM6-1", "585", "2061-2080", lon = c(16, 34), lat = c(-35, -22),
+                  var="bioc", res=0.5, path=path)
The geodata server appears to be down.
Hopefully it will be back soon.
download failed
